clusterloader2/pkg/measurement/common/network/performance-measurement-helper.go

```diff
@@ -0,0 +1,202 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package network
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"k8s.io/klog"
+	"k8s.io/perf-tests/clusterloader2/pkg/measurement"
+	"k8s.io/perf-tests/clusterloader2/pkg/util"
+	"math"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+)
+
+func (npm *networkPerformanceMeasurement) validate(config *measurement.Config) error {
+	var ratio, protocol string
+	var duration int
+	var err error
+
+	if duration, err = util.GetInt(config.Params, "duration"); err != nil {
+		return err
+	}
+
+	if ratio, err = util.GetString(config.Params, "ratio"); err != nil {
+		return err
+	}
+
+	if !checkRatioSeparator(ratio) {
+		return fmt.Errorf("ratio separator : missing")
+	}
+
+	podReplicas, err := util.GetInt(config.Params, "podReplicas")
+	if err != nil {
+		return err
+	}
+	if podReplicas <= 0 {
+		return fmt.Errorf("invalid value of podReplicas , should be > 0")
+	}
+
+	if ok, err := validateClientServerPodNum(ratio); !ok {
```

Comment:
Let's call it validateRatio maybe?

Reply:
Actually - it might be cleaner, if instead of using a ratio and making fancy validation, just make two separate parameters:
`numberOfServers`
`numberOfClients`

Reply:
Yes renamed it as ValidateRatio() and modified the logic a bit.